packages/frontend/faltest/smoke-test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const { setUpWebDriver } = require('@faltest/lifecycle');
+const Server = require('ember-cli-test-server');
+const assert = require('assert');
+
+describe('smoke', function() {
+  setUpWebDriver.call(this);
+
+  before(async function() {
+    this.server = new Server();
+
+    this.port = await this.server.start();
```

Comment:
does this default to the production build?

Reply:
It defaults to whatever is in your npm start script.

Reply:
ah. :-\ can it be changed? I think it'd be best to build for production and then run that from dist, yeah?

Reply:
possibly, but the reason it runs the script as a string is because any unique port is brought with it. Should we add `-prod` to the script?